### ISSUE TITLE
time-schedule fails when File is on NFS share

### ISSUE THREAD

**Issue by produnis** [robwebset/screensaver.video#1]
Hi there,
I manually set up 2 time-schedule entries.
However, these settings are just ignored.
Is there anything else I need to to in order to activate the time-schedule settings?

greetings
joe

**Comment by robwebset** [comment]
Please can you post a copy of your settings file (it's in userdata\addon_data\screensaver.video\settings.xml) and I'll see if there is anything obvious - and let me know what version of Kodi you are using (and the Skin)

Thanks, Rob

**Comment by robwebset** [comment]
One more thought, just to clarify - the time schedule is to instruct VideoScreensaver which video to use at which time of day.  It will not trigger the video when that time starts, it will just mean **IF** the screensaver starts during that time period, then it will use the selected video.

**Comment by produnis** [comment]
Hi Rob,
thx for replying quickly!!!

This is what I want:
- I have a folder containing screensaver-videos.
- From 6:am to 7pm, file "Aquarium" should be played IF screensaver is activated
- From 7pm to 6am, file "Fireplace" should be played IF screensaver is activated

This is what actually happens:
- If screensaver is activated, a random file from my screensaver-folder is played

This is my settings.xml:
```
<settings>
    <setting id="alterVolume" value="false" />
    <setting id="dimLevel" value="0" />
    <setting id="folderRepeatType" value="0" />
    <setting id="launchOnStartup" value="false" />
    <setting id="limitSessionToSingleCollection" value="false" />
    <setting id="logEnabled" value="false" />
    <setting id="mediaPlayingBlock" value="true" />
    <setting id="numberOfSchuleRules" value="2" />
    <setting id="overlayImage" value="0" />
    <setting id="overlayImageFile" value="" />
    <setting id="randomStart" value="false" />
    <setting id="resetStartupVolumeValue" value="90" />
    <setting id="resetVolumeOnStartup" value="false" />
    <setting id="rule1Day" value="0" />
    <setting id="rule1EndTime" value="18:59" />
    <setting id="rule1OverlayFile" value="" />
    <setting id="rule1OverlayImage" value="0" />
    <setting id="rule1StartTime" value="06:00" />
    <setting id="rule1VideoFile" value="nfs://192.168.0.123/Multimedia/VIDEO/JOE/XBMC/Screensaver/Aquarium.mp4" />
    <setting id="rule2Day" value="0" />
    <setting id="rule2EndTime" value="05:59" />
    <setting id="rule2OverlayFile" value="" />
    <setting id="rule2OverlayImage" value="0" />
    <setting id="rule2StartTime" value="19:00" />
    <setting id="rule2VideoFile" value="nfs://192.168.0.123/Multimedia/VIDEO/JOE/XBMC/Screensaver/Kaminfeuer.mp4" />
    <setting id="scheduleFile" value="" />
    <setting id="scheduleSource" value="1" />
    <setting id="screensaverFile" value="nfs://192.168.0.123/Multimedia/VIDEO/JOE/XBMC/Screensaver/Aquarium.mp4" />
    <setting id="screensaverFolder" value="nfs://192.168.0.123/Multimedia/VIDEO/JOE/XBMC/Screensaver/" />
    <setting id="screensaverFolderNested" value="true" />
    <setting id="screensaverVolume" value="0" />
    <setting id="showTime" value="true" />
    <setting id="showWeather" value="false" />
    <setting id="stopAfter" value="60" />
    <setting id="stopAutomatic" value="false" />
    <setting id="stopAutomaticShutdown" value="false" />
    <setting id="useAudioSuspend" value="false" />
    <setting id="useFolder" value="true" />
    <setting id="weatherAddon" value="weather.yahoo" />
</settings>

```

**Comment by robwebset** [comment]
Thanks for posting your config file - I have set my test system up in the same way and can not reproduce your problem. Please would you be able to enable debugging (Both system wide and in VideoScreensaver settings)

https://github.com/robwebset/screensaver.video/wiki#Debugging

Then post a link to the log file and I'll take a look and see if there is anything to give me a clue.

Thanks
Rob

**Comment by produnis** [comment]
This is my logfile:
http://pastebin.com/Qz8VbQN5
At line 48 it states "file not found". 
These files are there, and the Screensaver plays them well (see lines 109, 113, 117, 123, ). 
Only within schedule, it says "file not found".

**Comment by robwebset** [comment]
Thanks for the log file, I think the issue may be because the video file is on an NFS share - does it work if you have the file locally? If you can give it a go and let me know - then it highlights a possible fix for me which would be pretty easy for me to do.

Thanks
Rob

**Comment by produnis** [comment]
Hi rob 
I Put a file to my home directory and it works just fine
Seems you are right that it has something to do with the nfs share...

**Comment by robwebset** [comment]
Thanks for confirming that. I have done the change that I believe will fix this problem for you and pushed it to the repository, so you should be able to pull the update down to your installation.

Please can you let me know if this fixes it for you.

Fixed in: VideoScreensaver v100.6.0

**Comment by produnis** [comment]
Hi Rob,
I can confirm that with that new version everything works as expected.
THX A LOT!

greetings
Joe
